src/App.js

```diff
@@ -1,16 +1,67 @@
 import React from 'react';
-import TodoList from './TodoList';
-import AddTodoForm from './AddTodoForm';
+import {
+  BrowserRouter,
+  Routes,
+  Route
+} from "react-router-dom";
+import TodoList from './components/TodoList';
+import AddTodoForm from './components/AddTodoForm';
+import PropTypes from "prop-types";
 
+function TodoContainer() {
+  const [ todoList, setTodoList ] = React.useState([]);
+  const [ isLoading, setIsLoading ] = React.useState(true);
 
-function App() {
```

Comment:
I think there has been some misunderstanding, the function here should still be `App()`, the name of your functional component will almost always be the same as the file name. You mentioned you had some issues getting TodoContainer.js to work, but it's difficult for me to see the thinking behind this from just the code, so it would be best to discuss this with your tutor at the next opportunity.